.github/workflows/ci.yml

```diff
@@ -133,6 +133,12 @@ jobs:
           cd examples
           make -f MakefileExternal LIBTRIXI_PREFIX=$PWD/../install
 
+      - name: Test building in external CMake project
+        if: ${{ matrix.test_type == 'regular' }}
+        run: |
+          cd examples/external
+          ./build.sh
```

Comment:
Would it be possible to easily execute the built file here as well? I don't know, maybe this is overkill, but with all these library shenanigans we had, knowing it will not just build but also run would make me feel all warm and fuzzy inside 😅